app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  has_many :microposts, dependent: :destroy
```

Comment:
tạo quan hệ 1 - n, 1 user có thể có nhiều post,
dependent: tạo quan hệ phụ thuộc, khi 1 user bị xoá, toàn bộ bài viết của user đó cx bị xoá theo